src/components/ReportActionItem/ReportPreview.js

```diff
@@ -232,6 +237,15 @@ function ReportPreview(props) {
                                 style={[styles.requestPreviewBox]}
                             />
                         )}
+                        {isReportDraft && (
+                            <Button
+                                success
+                                medium
+                                text={translate('common.submit')}
+                                style={styles.requestPreviewBox}
+                                onPress={() => IOU.submitReport(props.iouReport)}
+                            />
+                        )}
```

Comment:
We should use the `isOwnPolicyExpenseChat` property of a workspace chat to determin if this is the employee or someonew else and if its their chat.

If `isOwnPolicyExpenseChat` is true then it shoudl be green like this:

<img width="494" alt="image" src="https://github.com/Expensify/App/assets/36083550/5206c155-ed35-433c-aa65-47219ea7a867">

For manager/ admin it should show like this 
<img width="503" alt="image" src="https://github.com/Expensify/App/assets/36083550/3107d4e9-2c18-47f8-9591-683969382d39">